src/main/java/teammates/storage/search/SearchManager.java

```diff
@@ -34,66 +54,151 @@ private boolean isSearchServiceActive() {
      */
     public StudentSearchResultBundle searchStudents(String queryString, List<InstructorAttributes> instructors)
             throws SearchNotImplementedException {
-        if (!isSearchServiceActive()) {
+        if (client == null) {
             throw new SearchNotImplementedException();
         }
-        // TODO
-        throw new SearchNotImplementedException();
+
+        QueryResponse response = null;
+        SolrQuery studentQuery = new SolrQuery();
+        studentQuery.setQuery("\"" + queryString + "\"");
```

Comment:
What happens if the query itself contains quotes?

Reply:
Two sets of double quotes negate each other as if there's no double quote. (e.g. 3 sets will behave like 1 set)

It will throw error if double quotes are imbalanced.